app/webpacker/components/RegistrationsV2/RegistrationAdministration/RegistrationActions.jsx

```diff
@@ -67,7 +67,7 @@ export default function RegistrationActions({
     .map((userId) => userEmailMap[userId])
     .join(',');
 
-  const changeStatus = (attendees, status) => {
+  const changeStatus = useCallback((attendees, status) => {
```

Comment:
I don't think this or the new function needs to be memoized.

Reply:
Changed